book/website/project-design/data-governance/data-gov-ml.md

```diff
@@ -0,0 +1,67 @@
+(pd-dg-ml)=
+
+# Data Governance for the Machine Learning Pipeline
+
+The Machine Learning (ML) pipeline consists of a series of activities including the collection of data, training of an ML model, and the deployment of the model into use. 
+Data is integral throughout the ML process and the methods for which data is collected, annotated, processed, and shared will impact individuals who may be the subjects in or creators of the data as well as communities represented in the data.
+
+This chapter will cover examples of data governance practices for ML for different steps in the pipeline such as: 
+- Data Collection
+- Data Management
+- Data Processing
+
+## Data Collection
+
+Many ML models are trained using datasets collected by the research team, which may be proprietary, or by using an open dataset that is available for download.
+The deep learning (DL) family of models, in particular, rely on massive corpuses of data such as text, code, images, sound, and other media.
+The process of data collection depends on the type and volume of data required and sources for acquisition.
+Many DL models rely on Internet data due to the sheer volume of digital content that is available on the web. 
+For example, [ImageNet datasets](https://www.image-net.org/about.php) are sourced from web images from image hosting websites like Flickr, and [LAION datasets](https://laion.ai/faq/) come from web crawling sources like Common Crawl.
+These methods of data collection through web scraping have raised issues regarding data quality and bias due to the nature of uncurated Internet data, as well as issues of data rights as web scraped data sets often include copyrighted data or data obtained without consent that may be in violation of the data license and terms of use.
```

Comment:
```suggestion
These methods of data collection through web scraping have raised issues regarding data quality and bias due to the nature of using uncurated Internet data, as well as issues of data rights as web scraped data sets often include copyrighted data or data obtained without consent that may be in violation of the data license and terms of use.
```

I'm not sure uncurated is the right word here, since pedantically any information on the internet has been curated (implicitly or explicitly) by someone.